src/test/java/core/basesyntax/FruitStorageServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import core.basesyntax.fruitservice.FruitStorage;
+import core.basesyntax.fruitservice.FruitStorageService;
+import core.basesyntax.fruitservice.Transaction;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitStorageServiceTest {
```

Comment:
Consider to make some fields in all tests constant, it is more convenient to everyone who read your code